src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -56,39 +64,67 @@
+  text: >
+    az account show --query user.name
+- name: Return results in plain text.
+  text: az account show --query MyTenantID --output tsv
+- name: Store the default subscription ID in a variable.
+  text: >
+    # Bash script
+    subscriptionId="$(az account show --query id --output tsv)"
+    echo "The current subscription ID is $subscriptionId"
 """
 
 helps['account set'] = """
 type: command
-short-summary: Set a subscription to be the current active subscription.
-"""
-
-helps['account show'] = """
-type: command
-short-summary: Get the details of a subscription.
-long-summary: If the subscription isn't specified, shows the details of the default subscription.
+short-summary: Change the active subscription.
+examples:
+- name: Change the active subscription using a subscription name
+  text: az account set --subscription "My subscription name"
+- name: Change the active subscription using a subscription ID
+  text: az account set --subscription MySubscriptionID
+- name: Change the active subscription using a variable.
+  text: >
+    # Bash script
+    subscriptionId="$(az account list --query "[?contains(name, 'OneOfMySubscriptionNames')].id" --output tsv)"
+    az account set --subscription $subscriptionId
+    echo "The current subscription has been set to $subscriptionId"
 """
 
 helps['account get-access-token'] = """
 type: command
 short-summary: Get a token for utilities to access Azure.
 long-summary: >
-    The token will be valid for at least 5 minutes with the maximum at 60 minutes.
+    The token will be valid for at least 5 minutes with a maximum of 60 minutes.
     If the subscription argument isn't specified, the current account is used.
 examples:
-    - name: Get an access token for the current account
-      text: >
-        az account get-access-token
-    - name: Get an access token for a specific subscription
-      text: >
-        az account get-access-token --subscription 00000000-0000-0000-0000-000000000000
-    - name: Get an access token for a specific tenant
-      text: >
-        az account get-access-token --tenant 00000000-0000-0000-0000-000000000000
-    - name: Get an access token to use with MS Graph API
-      text: >
-        az account get-access-token --resource-type ms-graph
+- name: Get an access token information for the current account.
+  text: az account get-access-token
+- name: Get an access token information for a specific subscription.
+  text: az account get-access-token --subscription MySubscriptionID
+- name: Get an access token information for a specific tenant.
+  text: az account get-access-token --tenant MySubscriptionID
+- name: Get an access token information to use with MS Graph API.
+  text: az account get-access-token --resource-type ms-graph
+- name: Get an access token information for a particular resource. If you receive a `Failed to connect to MSI...` error, your resource may not exist.
```

Comment:
`Failed to connect to MSI...` is a Cloud Shell bug, it is not related to non-existing resource.

```suggestion
- name: Get an access token information for a particular resource.
```